pages/All.html

```diff
@@ -29,10 +29,11 @@
   </div>
   <div class="coveo-main-section">
     <div class="coveo-facet-column">
-      <div class="CoveoFacet" data-title="Type" data-field="@objecttype" data-tab="All"></div>
-      <div class="CoveoFacet" data-title="FileType" data-field="@filetype" data-tab="All"></div>
-      <div class="CoveoFacet" data-title="Author" data-field="@author" data-tab="All"></div>
-      <div class="CoveoTimespanFacet" data-tab="All"></div>
```

Comment:
Won't people complain about losing the great timespanfacet ? Maybe it can be quickly created for DynamicFacetRange...

Reply:
Well... maybe, we'll see.

In any case, I much prefer we actually go forward and make it so the new facets are used everywhere *right now*, even if some stuff is still missing, and live with the consequence/fix missing stuff after the fact.

I feel like there's a some problem that will only appear when clients actually start using it at large and complain to us, and QA/internal pages won't cut it, so we have to bite the bullet.